airbyte-cdk/python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.11-alpine3.15 as base
+FROM python:3.9-slim as base
```

Comment:
Aren't we suppose to be able to avoid this as `RUN pip install --prefix=/install airbyte-cdk==<version>` would not require `pyarrow` anymore? I think that for this PR, it will fail since we don't update the version until there is an actual release but we can probably `approve-and-merge` bot this

Reply:
Else, we can set it to slim for this release and revert it to alpine afterward